app/views/posts/index.html.erb

```diff
@@ -0,0 +1,16 @@
+<div class="container-fluid posts">
+<div class="row">
+<div class="col-3 ">
+<%= render  "shared/new_post" %>
+</div>
+<div class="col-5 post-list">
+<div class="count text-center"> Recent posts <%=@posts.count%></div>
+<%@posts.each do |post|%>
```

Comment:
- [ ] **[OPTIONAL]** You can use [collection rendering](https://guides.rubyonrails.org/layouts_and_rendering.html#rendering-collections) to dry your views. Also please ident your code to make it more readable.